src/tree.js

```diff
@@ -139,6 +141,28 @@ define([
                 children = this.getChildren();
             callback(value, this.getID(), processChildren);
         },
+        getAbsolutePath: function (excludeRoot) {
+            if (this.isRootNode) {
+                if (excludeRoot) {
+                    return '';
+                }
+                return '/' + this.getID();
+            }
+            var mug = this.value, parentPath;
+            if (mug.p.dataParent) {
+                parentPath = mug.p.dataParent;
+            } else {
+                parentPath = this.parent.getAbsolutePath(excludeRoot);
+                if (parentPath === null) {
+                    return null;
+                }
+            }
+            var path = parentPath + '/' + this.getID();
+            if (mug.options.adjustPath) {
+                path = mug.options.adjustPath(mug, path);
```

Comment:
I suspect the rebase clobbered something or maybe omitted something. This should use `getPathName` instead of `adjustPath`.